File: php/examples/example_livedata.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $livedata = $client->liveData(['format' => 'xml']);
    $last_info = $client->getLastInfo();
    print_r($livedata);
    print_r($last_info);
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}

File: python/examples/example_livedata.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.live_data(format='xml'))
    print(v.get_last_info())
except ApiErrorException as e:
    print(e)
